docs/en_US/Compression/AutoPruningUsingTuners.rst

```diff
@@ -6,116 +6,63 @@ It's convenient to implement auto model pruning with NNI compression and NNI tun
+The previous example manually choosed L2FilterPruner and pruned with a specified sparsity. Different sparsity and different pruners may have different effect on different models. This process can be done with NNI tuners.
+
+Firstly, modify our codes for few lines
 
 .. code-block:: python
 
-   import nni
-   from nni.algorithms.compression.pytorch.pruning import *
-   params = nni.get_parameters()
-   conv0_sparsity = params['prune_method']['conv0_sparsity']
-   conv1_sparsity = params['prune_method']['conv1_sparsity']
-   # these raw sparsity should be scaled if you need total sparsity constrained
-   config_list_level = [{ 'sparsity': conv0_sparsity, 'op_name': 'conv0' },
-                        { 'sparsity': conv1_sparsity, 'op_name': 'conv1' }]
-   config_list_agp = [{'initial_sparsity': 0, 'final_sparsity': conv0_sparsity,
-                       'start_epoch': 0, 'end_epoch': 3,
-                       'frequency': 1,'op_name': 'conv0' },
-                      {'initial_sparsity': 0, 'final_sparsity': conv1_sparsity,
-                       'start_epoch': 0, 'end_epoch': 3,
-                       'frequency': 1,'op_name': 'conv1' },]
-   PRUNERS = {'level':LevelPruner(model, config_list_level), 'agp':AGPPruner(model, config_list_agp)}
-   pruner = PRUNERS(params['prune_method']['_name'])
-   pruner.compress()
-   ... # fine tuning
-   acc = evaluate(model) # evaluation
-   nni.report_final_results(acc)
+    import nni
+    from nni.algorithms.compression.pytorch.pruning import *
+   
+    params = nni.get_parameters()
+    sparsity = params['sparsity']
+    pruner_name = params['pruner']
+    model_name = params['model']
+
+    model, pruner = get_model_pruner(model_name, pruner_name, sparsity)
+    pruner.compress()
+
+    # after testing
+    nni.report_final_results(acc)
 
-Last, define our task and automatically tuning pruning methods with layers sparsity
+Then, define a ``config`` file in YAML to automatically tuning model, pruning algorithm and sparisty.
 
 .. code-block:: yaml
 
-   authorName: default
-   experimentName: Auto_Compression
-   trialConcurrency: 2
-   maxExecDuration: 100h
-   maxTrialNum: 500
-   #choice: local, remote, pai
-   trainingServicePlatform: local
-   #choice: true, false
-   useAnnotation: False
-   searchSpacePath: search_space.json
-   tuner:
-     #choice: TPE, Random, Anneal...
-     builtinTunerName: TPE
-     classArgs:
-       #choice: maximize, minimize
-       optimize_mode: maximize
-   trial:
-     command: bash run_prune.sh
-     codeDir: .
-     gpuNum: 1
+    searchSpace:
+    sparsity:
+      _type: choice
+      _value: [0.25, 0.5, 0.75]
+    pruner:
+      _type: choice
+      _value: ['slim', 'l2filter', 'fpgm', 'apoz']
+    model:
+      _type: choice
+      _value: ['vgg16', 'vgg19']
+    trainingService:
+    platform: local
+    trialCodeDirectory: .
+    trialCommand: python3 basic_pruners_torch.py --nni
+    trialConcurrency: 1
+    trialGpuNumber: 0
+    tuner:
+    name: grid
```

Comment:
the indent is strange

Reply:
better to tell users how to start this experiment

Reply:
Fix it